script/bootstrap

```diff
@@ -2,6 +2,5 @@
 
 ruby -v
 echo "==> Installing gems..."
-bundle check --path vendor/bundle 2>&1 > /dev/null || {
-  bundle install  --quiet --path vendor/bundle
-}
+bundle install --path vendor/bundle
```

Comment:
I believe this change was for debugging, but since we explained that, may as well revert it

Reply:
IMO this change is worth keeping - the github actions are going to start with a clean slate anyway so the bundle check should be redundant on the new build system

Reply:
Developers frequently use `script/ci_build` to run tests, which calls this. We don't need to see the full list of gems every time tests are re-run.

Reply:
👍